Dizzy/Screens/Discovery/PlacesInteractor.swift

```diff
@@ -33,6 +44,7 @@ class PlacesInteractor: PlacesInteractorType {
         webResourcesDispatcher.load(placesResource) {[weak self] result in
             switch result {
             case .success( let places):
+                self?.allPlaces.value = places
                 self?.delegate?.allPlacesArrived(places: places)
```

Comment:
as i mentioned above, we dont want to use delegate anymore

Reply:
Done